modules/webgl/src/adapter/resources/webgl-render-pipeline.ts

```diff
@@ -44,8 +44,6 @@ export class WEBGLRenderPipeline extends RenderPipeline {
   /** WebGL varyings */
   varyings: string[] | null = null;
 
-  _textureUniforms: Record<string, any> = {};
-  _textureIndexCounter: number = 0;
```

Comment:
Hmm, makes me uncertain, how are we tracking texture indices these days? each pipeline has a limited set of slots to play with and needs to distribute its textures over those?

Reply:
It looks like `_textureIndexCounter` was added in https://github.com/visgl/luma.gl/pull/1601 and has been unused since then, `_applyBindings()` appears to handle indices now. I haven't tested that area yet but hopefully if the glTF rendering is working again, multiple textures are supported!